bin/compile

```diff
@@ -178,43 +195,31 @@ if [ "$GIT_SSH_KEY" != "" ]; then
   ssh -oStrictHostKeyChecking=no -T git@github.com || true
 fi
 
+status "Pruning cached dependencies not specified in package.json"
+npm prune 2>&1 | indent
+
+status "Pruning cached bower dependencies not specified in bower.json"
+bower prune 2>&1 | indent
```

Comment:
my deployment breaks at this step with error
/tmp/buildpack_97da526d12730597e9c59e54eb19aba4/bin/compile: line 202: bower: command not found
 !     Push rejected, failed to compile Ember CLI app